src/main/kotlin/com/kotlin/boardproject/domain/notification/service/SseServiceImpl.kt

```diff
@@ -1,11 +1,11 @@
-package com.kotlin.boardproject.service
+package com.kotlin.boardproject.domain.notification.service
 
-import com.kotlin.boardproject.common.util.log
-import com.kotlin.boardproject.dto.notification.NotificationResponseDto
-import com.kotlin.boardproject.model.Notification
-import com.kotlin.boardproject.repository.NotificationRepository
-import com.kotlin.boardproject.repository.UserRepository
-import com.kotlin.boardproject.repository.common.SseRepository
+import com.kotlin.boardproject.global.util.log
```

Comment:
🚫 **[ktlint]** <standard:import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between with "java", "javax", "kotlin" and aliases in the end